src/components/blogposts/BlogTeaser.js

```diff
@@ -1,9 +1,22 @@
+import {useState} from 'react';
 import styled from 'styled-components';
 
 export default function BlogTeaser() {
+	const [{data, error}, setPosts] = useState({data: '', error: null});
```

Comment:
you can use the useFetch hook or you just use posts instead of data and error, because you have no catch block and so your error will always be null.

Reply:
as mentioned, I switched to the useFetch Hook